### ISSUE TITLE
Traveling to jupiter in mother ship traps you forever in space

### ISSUE THREAD

**Issue by Cjm5001** [katzenpapst/amunra#36]
When I traveled in a mothership to jupiter, i tried using the shuttle rocket to get out and when it got real high, it kicked me out and i was stuck in mid-air.

**Comment by katzenpapst** [comment]
So, you were a) in transit, b) started a shuttle, and c) were kicked from the server when the selection gui should have appeared?

**Comment by Cjm5001** [comment]
I traveled from earth to jupiter, tried to leave, and i was stuck. so a.

**Comment by katzenpapst** [comment]
No, that wasn't a "pick one" list. I meant: did these 3 things happen in that order? Let me try again:

So, you had a client/server setup, not just singleplayer. You were on a mothership. You started transit to jupter. While in transit, you got into a shuttle and started it. Then, upon reaching the exit height, you got kicked off the server. When you rejoined, you found yourself stuck at exit height, unable to move.

Correct?

**Comment by Cjm5001** [comment]
I was in singleplayer, when I traveled to Jupiter, I tried getting into the shuttle (Already at Jupiter, not in transit), then when I got high up from the mothership, it kicked me out of the rocket and I was stuck in mid air. I tried /kill but it sent me back to the mothership.

**Comment by katzenpapst** [comment]
So the problem is that the star selection screen didn't open for you on reconnect. It should recognize that you are in the "stuck in mid air" state, and show the screen. I thought I fixed this one, I guess I have to fix it better.

**Comment by Cjm5001** [comment]
Thanks for the help!

**Comment by katzenpapst** [comment]
Hmm, do you have any crash report from the time you were kicked? Also, are you sure you weren't able to float back down to the mothership from the mid-air position? 

I can't reproduce this, and I don't see anything which might cause this...

**Comment by Cjm5001** [comment]
The game didn't crash, it just kicked me out of the rocket and I was stuck in mid air.

**Comment by katzenpapst** [comment]
Oh. sure you didn't accidentally dismount?

And, usually you should have been able to float down back to the mothership. The rocket itself should also be able to recognize a lack of player and enter landing mode, too. So, either that didn't work, or the selection screen didn't open for some reason. Do you have the logfile from that time anyway? Also, what happens if you reload that save? if you are still mid-air, can you press F3 and then try to fly to 0/64/0?
